ts/src/index.ts

```diff
@@ -152,9 +171,24 @@ function logError(msg: string, config: Config) {
  * profiles.
  */
 export async function startLocal(config: Config = {}): Promise<void> {
+  // Start the v8 heap profiler if heap profiling is enabled. This must be done
```

Comment:
It feels that code from here up to "profiler = new Profiler(normalizedConfig);" could be shared between start() and startLocal()?